test/smoke/testApps/TraceLog4j1_2/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/TraceLog4j1_2Test.java

```diff
@@ -0,0 +1,58 @@
+package com.microsoft.applicationinsights.smoketest;
+
+import static org.junit.Assert.assertEquals;
+
+import com.microsoft.applicationinsights.internal.schemav2.MessageData;
+import com.microsoft.applicationinsights.internal.schemav2.SeverityLevel;
+
+import org.junit.Assume;
+import org.junit.Test;
+
+public class TraceLog4j1_2Test extends AiSmokeTest {
+    @Test
+    @TargetUri("/traceLog4j1_2")
+    public void testTraceLog4j1_2() {
+        // this doesn't work with jbosseap6; under investigation
+        Assume.assumeFalse(currentImageName.contains("jbosseap6"));
+
+        assertEquals(1, mockedIngestion.getCountForType("RequestData"));
```

Comment:
Same thing here. Since the filter isn't under test in this class, we don't need to check for RequestData. (but again, it doesn't hurt 😄 )